core/audits/byte-efficiency/byte-efficiency-audit.js

```diff
@@ -342,7 +343,7 @@ class ByteEfficiencyAudit extends Audit {
       }
 
       metricSavings.FCP = fcpSavings;
-      metricSavings.LCP = Math.max(lcpGraphSavings, lcpRecordSavings);
+      metricSavings.LCP = Math.max(lcpSavings, lcpRecordSavings);
```

Comment:
Ope my bad. Forgot this was `lcpGraphSavings` before. `lcpGraphSavings` is better.